domain/src/main/java/com/personalizatio/domain/usecases/preferences/GetPreferencesValueUseCase.kt

```diff
@@ -0,0 +1,21 @@
+package com.personalizatio.domain.usecases.preferences
+
+import com.personalizatio.domain.repositories.PreferencesRepository
+import javax.inject.Inject
+
+class GetPreferencesValueUseCase @Inject constructor(
+    private val preferencesRepository: PreferencesRepository
+) {
+
+    fun getSidLastActTime(defaultValue: Long = 0) = preferencesRepository.getSidLastActTime(defaultValue)
```

Comment:
например тут должно быть
```suggestion
    fun getSidLastActTime(defaultValue: Long = 0L): Long = preferencesRepository.getSidLastActTime(defaultValue)
```

Reply:
Изменила